### ISSUE TITLE
feat: suppress 'no-underscore-dangle' when using R.__

### ISSUE THREAD

**Issue by hitmands** [ramda/eslint-plugin-ramda#25]
This is a **Feature Request**

## Actual
[`R.__`](https://ramdajs.com/docs/#__) causes [`no-underscore-dangle`](https://eslint.org/docs/rules/no-underscore-dangle) eslint error.

## Desired
Ramda placeholder shouldn't cause this error.

## Current Workaround
```javascript
/* eslint no-underscore-dangle: ["error", { "allow": ["__"] }] */

const postProp = R.prop(R.__, { id: 2, title: 'Hello World', author: 'Giuseppe Mandato'});
```

**Comment by haskellcamargo** [comment]
The `no-underscore-dangle` is a core [eslint rule](https://eslint.org/docs/rules/no-underscore-dangle) as far as I know. I don't know if there is much that we can do on `eslint-plugin-ramda`. @lo1tuma, opinions on this?
